Library/Formula/plan9port.rb

```diff
@@ -1,6 +1,7 @@
 class Plan9port < Formula
   desc "Many Plan 9 programs ported to UNIX-like operating systems"
   homepage "https://swtch.com/plan9port/"
+  head "https://github.com/9fans/plan9port.git"
```

Comment:
Never mind I read [the PR guide](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md#how-to-open-a-homebrew-pull-request-and-get-it-merged) and squashed it.